test_scripts/Smoke/Resumption/commonStepsResumption.lua

```diff
@@ -0,0 +1,71 @@
+local commonStepsResumption = {}
```

Comment:
@LitvinenkoIra, scripts: 
Smoke/Resumption/ATF_Resumption_app_registered_in_more_than_30sec_after_IGN_OFF.lua 
Smoke/Resumption/ATF_Resumption_disconnect_more_than_30s_before_IGN_OFF.lua 
still using  test_scripts/Smoke/Resumption/, please update.

After update remove not used common script.